deployment/GI05MOAS-GL537_Deploy.csv

```diff
@@ -1,5 +1,5 @@
 CUID_Deploy,deployedBy,CUID_Recover,recoveredBy,Reference Designator,deploymentNumber,versionNumber,startDateTime,stopDateTime,mooring.uid,node.uid,sensor.uid,lat,lon,orbit,deployment_depth,water_depth,notes,electrical.uid,assembly_template_revision
-AR60-01,,,,GI05MOAS-GL537-00-ENG000000,3,1,2021-08-15T00:00:00,,CGVGI-05MOAS-00000,CGVEH-GLDROO-00537,CGCON-GLDROO-00537,59.925833,-39.299167,,1000,1000,,,
-AR60-01,,,,GI05MOAS-GL537-01-FLORDM000,3,1,2021-08-15T00:00:00,,CGVGI-05MOAS-00000,CGVEH-GLDROO-00537,CGINS-FLORDM-03582,59.925833,-39.299167,,1000,1000,,,
-AR60-01,,,,GI05MOAS-GL537-02-DOSTAM000,3,1,2021-08-15T00:00:00,,CGVGI-05MOAS-00000,CGVEH-GLDROO-00537,CGINS-DOSTAM-00401,59.925833,-39.299167,,1000,1000,,,
-AR60-01,,,,GI05MOAS-GL537-04-CTDGVM000,3,1,2021-08-15T00:00:00,,CGVGI-05MOAS-00000,CGVEH-GLDROO-00537,CGINS-CTDGVM-09237,59.925833,-39.299167,,1000,1000,,,
\ No newline at end of file
+AR60-01,,,,GI05MOAS-GL537-00-ENG000000,3,1,2021-08-11T04:10:00,,CGVGI-05MOAS-00000,CGVEH-GLDROO-00537,CGCON-GLDROO-00537,59.85909,-39.12473,,1000,1000,,,
+AR60-01,,,,GI05MOAS-GL537-01-FLORDM000,3,1,2021-08-11T04:10:00,,CGVGI-05MOAS-00000,CGVEH-GLDROO-00537,CGINS-FLORDM-03582,59.85909,-39.12473,,1000,1000,,,
+AR60-01,,,,GI05MOAS-GL537-02-DOSTAM000,3,1,2021-08-11T04:10:00,,CGVGI-05MOAS-00000,CGVEH-GLDROO-00537,CGINS-DOSTAM-00401,59.85909,-39.12473,,1000,1000,,,
+AR60-01,,,,GI05MOAS-GL537-04-CTDGVM000,3,1,2021-08-11T04:10:00,,CGVGI-05MOAS-00000,CGVEH-GLDROO-00537,CGINS-CTDGVM-09237,59.85909,-39.12473,,1000,1000,,,
```

Comment:
For Lat I get 59.85908333...so 59.85908. The rest looks good

Reply:
Concur with @rtravis-whoi.  Perhaps the 59.95909 is the original from which 59 51.525 was calucated?